wiki/Glossary/en.md

```diff
@@ -967,26 +890,22 @@ However, if you have an [osu!supporter](/wiki/osu!supporter/) tag, the beatmapse
 
 Spectator aura is a mysterious phenomenon that could cause players to choke their combo, or fail a beatmap from having a spectator(s) spectate their gameplay.
 
-### Spinner Spam
-
-Spinner spam is a **heavily discouraged** mapping technique (in fact, it will probably make your map unrankable).
-
-As the name sugguests, spinner spam occurs when a mapper spams many very short spinners (that last for ~2 sec or less) one after another.
-On the other hand, there is the inverse of this, very long spinners that are placed right after another.
-
-Before considering this mapping technique, it is very important to note:
-
-- Spinners can cause fatigue to the player, since it requires rapid, repetitive, and vigorous motion.
-- The circular motion of a spinner will cause the cursor to be placed in random locations after completion.
-  - This will confuse the player when they are needed to click the next object(s).
-
 ### Stacked
 
 Stacked objects are hit objects that overlap each other in the playfield.
 The most common object that is stacked are hit circles.
 
 Stacking hit circles (a clear violation of beat spacing) is, nevertheless, permitted if the hit circles are very close to each other in the timeline (half a beat or usually even less).
 
+### Standard
+
+_See also: [osu!standard](/wiki/osu!standard)._
```

Comment:
I was like 10 seconds lazy... :s but yeah this is a redirect-- I will write specs on it after #36 is done